pybop/optimisers/pints_optimisers.py

```diff
@@ -7,7 +7,7 @@ class GradientDescent(pints.GradientDescent):
     """
 
     def __init__(self, x0, sigma0=0.1, bounds=None):
-        boundaries = PintsBoundaries(bounds, x0)
+        boundaries = None  # Bounds ignored in pints.GradDesc
```

Comment:
Is it safe to overwrite the bounds input here? Should a message be returned to the user that bounds are being ignored?